languages/csharp/exercises/concept/enums-advanced/.meta/design.md

```diff
@@ -0,0 +1,69 @@
+# Design
```

Comment:
It might be nice to add this document to the sidebar on the website in addition to the [exercise overview](https://exercism.github.io/v3/#/languages/csharp/exercises/concept/README)

Reply:
We might want to refer to the design document in the exercise overview, as that link is currently broken for the website. I'll also update the sidebar.

Reply:
@SaschaMann I've updated.

Reply:
LGTM